src/pages/portfolio/AccountDetailsAndHistory.tsx

```diff
@@ -51,7 +50,7 @@ const usePortfolioValues = ({
             selectedLocale,
             dateFormat: 'medium',
           })
-        : stringGetter({ key: STRING_KEYS.PORTFOLIO_VALUE }),
+        : stringGetter({ key: STRING_KEYS.TRADING_ACCOUNT }),
```

Comment:
interesting- just to confirm, we're fine with these changes going out now? instead of being flagged behind either the ui or vaults flag?

Reply:
the empty state looks sorta funny - any chance we can make it look less empty? (ie having an empty bar
<img width="959" alt="Screenshot 2024-10-03 at 11 05 36 AM" src="https://github.com/user-attachments/assets/001fa303-07eb-4859-b090-65757387bc60">

Reply:
Might need to go over empty state with designers here

Reply:
hmmm I don't think an empty bar makes sense here. I honestly don't hate the empty state as it is

Reply:
We hide on tablet on purpose, it doesn't really add much for small screens imo